### PR TITLE
remove usage of deprecated ptypes, fix linter issues

### PR DESCRIPTION
**Pull request by majst01**

### CODE REVIEW

**Review by Gerrit91 — APPROVED** [review]
Looks good. Should we verify this is working in the mini-lab before merging?

**Comment by majst01** [comment]
maybe just a image override in test is easier

**Review by droid42 — APPROVED** [review]
LGTM

**Comment by majst01** [comment]
tested in test environment with success (project, tenant ls, billing container etc)

File: api/rest/mapper/mapper.go [hunk 1]
```diff
@@ -234,43 +232,19 @@ func ToV1Quota(q *mdmv1.Quota) *v1.Quota {
...
-}
-
-func timeToTimestamp(t *time.Time) (*timestamp.Timestamp, error) {
-	if t == nil {
-		return nil, nil
-	}
-
-	return px.TimestampProto(*t)
+func mustTimeToTimestamp(t *time.Time) *timestamppb.Timestamp {
+	return timestamppb.New(*t)
```
Comment (majst01) [inline api/rest/mapper/mapper.go:249]: nil check

File: api/rest/mapper/mapper.go [hunk 2]
```diff
@@ -234,43 +232,19 @@ func ToV1Quota(q *mdmv1.Quota) *v1.Quota {
...
-
-func timestampToTime(ts *timestamp.Timestamp) (*time.Time, error) {
-	if ts == nil {
-		return nil, nil
-	}
-
-	t, err := px.Timestamp(ts)
-	return &t, err
+func mustTimestampToTime(ts *timestamppb.Timestamp) *time.Time {
+	t := ts.AsTime()
```
Comment (majst01) [inline api/rest/mapper/mapper.go:244]: nil check

**Pull request merged**